site/content/docs/user/local-registry.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Local Registry"
+menu:
+  main:
+    parent: "user"
+    identifier: "user-local-registry"
+    weight: 3
+---
+# Local Registry
+
+With kind v0.6.0 there is a new config feature `containerdConfigPatches` that can
+be leveraged to configure insecure registires.
+The following recipe leverages this to enable a localregistry.
+
+## Create A Cluster And Registry
+
+The following shell script will create a local docker registry and a kind cluster
+with it enabled.
+
+```
+#!/bin/sh
+set -o errexit
+
+# create registry container unless it already exists 
+REGISTRY_CONTAINER_NAME='kind-registry}'
+REGISTRY_PORT='5000'
+if [ "$(docker inspect -f '{{.State.Running}}' "${REGISTRY_CONTAINER_NAME}")" != 'true' ]; then
+  docker run -d -p "${REGISTRY_PORT}:5000" --restart=always --name "${REGISTRY_CONTAINER_NAME}" registry:2
+fi
+
+# create a cluster with the local registry enabled in containerd
+cat <<EOF | kind create cluster --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+containerdConfigPatches: 
+- |-
+  [plugins."io.containerd.grpc.v1.cri".registry.mirrors."registry:${REGISTRY_PORT}"]
+    endpoint = ["http://registry:${REGISTRY_PORT}"]
+EOF
+
+# add the registry to /etc/hosts on each node
+for node in $(kind get nodes); do
+  docker exec "${node}" sh -c "echo $(docker inspect --format '{{.NetworkSettings.IPAddress }}' "${REGISTRY_CONTAINER_NAME}") registry >> /etc/hosts"
+done
+```
+
+## Using The Registry
+
+The registry can be used like this.
+
+1. First we'll pull an images `docker pull docker pull gcr.io/google-samples/hello-app:1.0`
+2. Then we'll tag the image to use the local registry `docker tag gcr.io/google-samples/hello-app:1.0 localhost:5000`
```

Comment:
Previously you made 5000 a variable in REGISTRY_PORT

Reply:
it's not overridable though, that was just to make the script refactorable

Reply:
the command is wrong, it should something like `docker tag gcr.io/google-samples/hello-app:1.0 localhost:5000/hello-app:1.0`